### DISCUSSION TITLE
Card import of CSV exported from Dashlane misses validity month

### DISCUSSION THREAD

**Discussion by eleaner** [dani-garcia/vaultwarden#3020] [Q&A]
### Subject of the issue
Card import of CSV exported from Dashlane misses validity month

### Deployment environment

* Install method: docker

* Clients used: web client

* Reverse proxy and version: traefik

* MySQL/MariaDB or PostgreSQL version: n/a

* Other relevant details:

### Steps to reproduce
<img width="680" alt="image" src="https://user-images.githubusercontent.com/5169203/208735296-768d6523-1c84-446e-8978-809adcf2f5d1.png">

[test.csv](https://github.com/dani-garcia/vaultwarden/files/10270988/test.csv)
try uploading attached file

### Expected behaviour
Expiration month is loaded from file

### Actual behaviour
<img width="827" alt="image" src="https://user-images.githubusercontent.com/5169203/208734840-2e1cff64-0a78-45b5-8a95-b5208579518b.png">
Expiration month is not loaded while in the file is "03"
<img width="160" alt="image" src="https://user-images.githubusercontent.com/5169203/208734966-9a472c2f-5de3-438c-ab48-01c60eee8acb.png">

**Comment by BlackDex** [comment]
Imports are all handled client side. This is not something the Vaultwarden developers maintain.

There doesn't seem to be a ticket about it. If you can replicate it on vault.bitwarden.com, i would suggest to report an issue there.
